prime-router/build.gradle.kts

```diff
@@ -454,7 +455,19 @@ tasks.register("quickPackage") {
     tasks["dokkaHtml"].enabled = false
 }
 
+/**
+ * Docker services needed for running Dockerless
+ */
+dockerCompose {
+    projectName = "prime-router"
+    useComposeFiles.addAll("docker-compose.yml")
+    startedServices.addAll("sftp", "ftps", "soap-webservice", "vault", "azurite")
+    stopContainers.set(false)
+    waitForTcpPorts.set(false)
+}
+
 tasks.azureFunctionsRun {
+    dependsOn("composeUp")
```

Comment:
Required the postgres server running to run automatically with `./gradlew run` on a fresh install

Reply:
Resolved from external conversation. In order for this to be meaningful it would have to be spun up for the `build` step which causes all sorts of odd results.